scripts/world_bank/datasets/datasets.py

```diff
@@ -344,13 +561,13 @@ def sanitize_csv_key(key):
     return re.sub(r'[^a-zA-Z0-9]', '', key).lower()
 
 
-def get_series_file_name(zip):
+def get_data_and_series_file_names(zip):
     for file_name in zip.namelist():
         if file_name.endswith(DATA_FILE_SUFFIX):
             series_file_name = file_name.replace(DATA_FILE_SUFFIX,
                                                  SERIES_FILE_SUFFIX)
             if series_file_name in zip.namelist():
-                return series_file_name
+                return [file_name, series_file_name]
```

Comment:
can return a tuple: `return (file_name, series_file_name)` or `return (None, None)`
caller can assign to a tuple:
```(file_name, series) = get_data_and_series_file_names(zip)
  if file_name is not None:
    ...
```

Reply:
That's a good idea. Done.